### DISCUSSION TITLE
Multi langue content [Aug 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6171] [Discord Threads]
> 🙋‍♂️ **.berteh**

**Comment by o1lab** [comment]
> 🙋‍♂️ **.berteh**  
#### Bump: any idea how I could handle multi langue content? Or set a display value that might change according to some users preference (in this case language)?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1116429534561054801)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
